src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAvailabilityEventArgs.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace System.Net.NetworkInformation
+{
+    public class NetworkAvailabilityEventArgs : EventArgs
+    {
+        private bool _isAvailable;
```

Comment:
Ah: actually this is a problem: you seem to be using an older version of the Windows code. 
Prior to splitting files, please ensure you have synced-up with what's on GitHub corefx/master:

In `NetworkAddressChange.cs`: 
```C#
 public class NetworkAvailabilityEventArgs : EventArgs
    {
        private readonly bool _isAvailable;

        internal NetworkAvailabilityEventArgs(bool isAvailable)
```

We had the same issue with S.N.Security where valuable PR feedback may have been lost and now we need to re-add all the fixes.